packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -183,9 +189,10 @@
             kind="default"
             importance="primary"
             :class="[
+              'dt-message-input__button',
```

Comment:
Sorry I messed up with my suggestion previously, because this will also affect the case where there's text in the button:
<img width="120" alt="image" src="https://github.com/user-attachments/assets/4c925512-92c0-403f-a71a-0ab4eb82570e">
You could add a class for the case where `showSendIcon` is true, to set the `max-width`, and have `dt-message-input__button` for only setting `max-height` and `border-radius`.

Reply:
@ninamarina I'm not quite sure what you're getting at

Reply:
Yes, that's a possibility. And then you would need another class for all of the buttons.